src/FakeItEasy.Analyzer.Shared/FakeItEasy.Analyzer.Shared.projitems

```diff
@@ -12,6 +12,7 @@
     <PackageTags>FakeItEasy;analyzer</PackageTags>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)FakeItEasy.Analyzer.ruleset</CodeAnalysisRuleSet>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>

```

Comment:
I was about to do some ugly hack with private targets, and then I found this property, thanks to MSBuild Structured Log Viewer. Today, I love MSBuild (tomorrow I'll probably hate it again!)